src/components/Button/LikeBox.tsx

```diff
@@ -28,12 +28,12 @@ export default React.memo(function LikeBox(props: LikeBoxProps) {
     >
       <Icon
         name="thumbs up outline"
-        className={active ? "button-box__icon" : "button-box__icon-hovered"}
+        className={!active ? "button-box__icon" : "button-box__icon-hovered"}
       />
 
       <Icon
         name="thumbs up"
-        className={!active ? "button-box__icon" : "button-box__icon-hovered"}
+        className={active ? "button-box__icon" : "button-box__icon-hovered"}
```

Comment:
Idem to `FavoriteBox`

```suggestion
        className={TokinList.join([
          "button-box__icon",
          !!active && "button-box__icon--active"
        ])}
```